frontends/verific/verific.cc

```diff
@@ -2244,7 +2244,7 @@ struct VerificPass : public Pass {
 					continue;
 				}
 				if (args[argidx] == "-L" && argidx+1 < GetSize(args)) {
-					verific_sva_fsm_limit = atoi(args[++argidx].c_str());
+					verific_sva_fsm_limit = std::stoi(args[++argidx]);
```

Comment:
This can throw an exception that we do not catch (instead of returning zero). Is this really the behavior we want?

Reply:
Ack. Good point. I didn't realise `stoi` could raise an exception (which is arguably better than silently returning zero) but let's not get into that rabbit hole right now.